collections/_article/godot-4-0-sets-sail.md

```diff
@@ -0,0 +1,685 @@
+
+<a id="multithreading-performance-optimization"></a>
+### Multithreading & Performance Optimization
+
+On the performance front, techniques such as broadphase optimization and multithreading were implemented for both 2D and 3D environments. Depending on your scene, the number of physics bodies, and the number of CPU cores, this can lead to a great increase in simulation speed. Some of these improvements have already been backported to the latest Godot 3 releases.
+
+<a id="better-physics-api"></a>
+### Better Physics API
+
+With that done, it was time to improve the user side of things. We took the opportunity to carry out [a major reorganization of physics nodes](https://github.com/godotengine/godot/pull/48908) and improve many APIs/behaviors to make the experience more user-friendly ([collision layers logic](https://github.com/godotengine/godot/pull/50625), [RigidBodies](https://github.com/godotengine/godot/pull/55736), etc.). A lot of properties previously unique to specific body types are now available to all **PhysicsBody** nodes. Also, the new **CharacterBody** node has now replaced old kinematic bodies for enhanced behavior in [2D](https://github.com/godotengine/godot/pull/51027/files) and [3D](https://github.com/godotengine/godot/pull/52889). This allows you to have an advanced character controller ready to use with new configurable properties for flexibility.
+
+Scripting properties is simpler now as well. In previous engine versions, properties related to moving, sliding, and colliding had to be manually passed to move_and_slide(). They can now be set up on the nodes themselves using scenes. This reduces the amount of code needed for desired physical interactions. In addition, physics layers and masks have been made more intuitive.
+
+![](/storage/blog/godot-4-0-sets-sail/06-physics-better-character-body.png)
+
+<a id="higher-simulation-stability"></a>
+### Higher Simulation Stability
+
+A new release is not just new features. A significant effort went into fixing previous issues causing jitter and imprecise computations. So you can look forward to higher simulation stability. For more on this effort, contributors Camille Mohr-Daurat ([pouleyKetchoupp](https://github.com/pouleyKetchoupp)), [lawnjelly](https://github.com/lawnjelly), and Fabrice Cipolla ([fabriceci](https://github.com/fabriceci)) have documented their work in [this blog post](https://godotengine.org/article/physics-progress-report-1).
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/godot-4-0-sets-sail/06-physics-kicking-ball.mp4" type="video/mp4">
+</video>
+
+<a id="remarks"></a>
+### Remarks
```

Comment:
```suggestion
**Important remarks:**
```

Remarks are formatted inconsistently, bold text without a heading is IMO the best option out of available. (This also affects TOC)